### PR TITLE
Remove obsolete code handling of file scheme for importUri.

### PR DESCRIPTION
**Pull request by terrylucas**
Fixes https://github.com/flutter/flutter/issues/26615
Requires Flutter Dev Release 1.4.12 or newer that has the VM change.

### CODE REVIEW

File: packages/devtools/lib/src/inspector/diagnostics_node.dart [hunk 1]
```diff
@@ -683,38 +683,22 @@ class InspectorSourceLocation {
 
   String get path => JsonUtils.getStringMember(json, 'file');
 
-  String getFile() {
-    final fileName = path;
-    if (fileName == null) {
-      return parent != null ? parent.getFile() : null;
```
Comment (jacob314) [inline packages/devtools/lib/src/inspector/diagnostics_node.dart]: this part actually shouldn't be removed. it is falling back to the file path specified in the parent for the case of nested location objects (that feature of tracking the locations for parameters that you are looking at testing the performance benefit of removing).

File: packages/devtools/lib/src/inspector/diagnostics_node.dart [hunk 2]
```diff
@@ -683,38 +683,22 @@ class InspectorSourceLocation {
...
-    }
-
-    // We have to strip the file:// or file:/// prefix depending on the
-    // operating system to convert from paths stored as URIs to local operating
-    // system paths.
-    // TODO(jacobr): remove this workaround after the code in package:flutter
-    // is fixed to return operating system paths instead of URIs.
-    // https://github.com/flutter/flutter-intellij/issues/2217
-    return fromSourceLocationUri(fileName);
```
Comment (jacob314) [inline packages/devtools/lib/src/inspector/diagnostics_node.dart]: from SourceLocationUri on the other hand can be removed with your change.

File: packages/devtools/lib/src/inspector/inspector_service.dart [hunk 3]
```diff
@@ -865,27 +865,6 @@ class ObjectGroup {
   }
 }
 
-// TODO(jacobr): can we get the host OS from VMService.
```
Comment (jacob314) [inline packages/devtools/lib/src/inspector/inspector_service.dart]: great to see this removed. it is broken in devtools anyway. :)

File: packages/devtools/lib/src/inspector/diagnostics_node.dart (hunk 1 above)
Comment (terrylucas) [reply packages/devtools/lib/src/inspector/diagnostics_node.dart]: Oops, too aggressive put back getFile and useages.

File: packages/devtools/lib/src/inspector/inspector_service.dart (hunk 3 above)
Comment (terrylucas) [reply packages/devtools/lib/src/inspector/inspector_service.dart]: :-)

**Comment by terrylucas** [comment]
PTAL getFile and usage are back.

**Pull request merged**